tuxemon.py

```diff
@@ -43,16 +43,16 @@
 
 if __name__ == '__main__':
     parser = ArgumentParser()
-    parser.add_argument('-d', '--data', dest='data', metavar='resources', type=str, nargs='?',
-                        default=None, help='The data directory to use')
+    parser.add_argument('-g', '--game', dest='game', metavar='tuxemon', type=str, nargs='?',
```

Comment:
@ShadowApex @xirsoi @kerizane 
This is a proposal to add another folder for looking up game resources.  Historically we have been using the term "campaign".  tbh, I'm not really a fan for using the word "game" here.  could i get your opinions on what we should be calling this feature?

Reply:
For now I'm reverting it back to data which was the old name. I believe it can be altered in the future.